core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -21,13 +21,26 @@
  */
 public class ClientConfigProperties extends AbstractConfig {
 
+    /**
+     * The default amount of time to wait for credits after link creation.
+     */
+    public static final long DEFAULT_FLOW_LATENCY = 20L; //ms
+    /**
+     * The default number of credits issued by the receiver side of a link.
+     */
+    public static final int  DEFAULT_INITIAL_CREDITS = 200;
+    /**
+     * The default amount of time to wait for a response before a request times out.
+     */
+    public static final long DEFAULT_REQUEST_TIMEOUT = 200L; // ms
+
     private String name;
     private String host = "localhost";
     private int port = Constants.PORT_AMQPS;
     private String username;
     private char[] password;
     private String amqpHostname;
-    private long waitMillisForCredits = 10L;
+    private long flowLatency = 10L;
     private int initialCredits = 1000;
     private long requestTimeoutMillis = 200L;
```

Comment:
I think your newly created static defaults should be assigned here for flowLatency, initialCredits and requestTimeoutMillis.

Reply:
absolutely, I simply forgot to do that. Thanks for spotting :+1: